packages/node_modules/@webex/plugin-meetings/src/media/properties.js

```diff
@@ -25,8 +25,8 @@ export default class MediaProperties {
     this.remoteShare = options.remoteShare;
     this.remoteAudioTrack = options.remoteAudioTrack;
     this.remoteVideoTrack = options.remoteVideoTrack;
-    this.localQualityLevel = options.localQualityLevel || QUALITY_LEVELS.HIGH;
-    this.remoteQualityLevel = options.remoteQualityLevel || QUALITY_LEVELS.HIGH;
+    this.localQualityLevel = options.localQualityLevel || QUALITY_LEVELS['480p'];
```

Comment:
Using default as 480p for local and remote, As per this JIRA ticket -https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-351410

Reply:
720p for local